### PR TITLE
Plumb table stats for AggregationNodes

### PR DESCRIPTION
**Pull request by aaneja**
This will record these table stats in QueryCompletedEvent. This is helpful for understanding how/why partial aggregation was done for a query

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by ClarenceThreepwood** [comment]
Looks like the test failures are related

File: presto-main/src/main/java/com/facebook/presto/sql/planner/InputExtractor.java [hunk 1]
```diff
@@ -107,6 +108,14 @@ public Void visitSpatialJoin(SpatialJoinNode node, Context context)
             return null;
         }
 
+        @Override
+        public Void visitAggregation(AggregationNode node, Context context)
+        {
+            context.setExtractStatistics(true);
```
Comment (ClarenceThreepwood) [inline presto-main/src/main/java/com/facebook/presto/sql/planner/InputExtractor.java:114]: I'm just curious - what is the rationale behind not extracting stats for some plan nodes? Too much noise?

Comment (aaneja) [reply presto-main/src/main/java/com/facebook/presto/sql/planner/InputExtractor.java:114]: Yes, IMO that was the reason. I mostly agree - join & aggs are primary use cases where table stats would be very helpful for debugging behavior

**Comment by aaneja** [comment]
> Looks like the test failures are related

Fixed now.

**Review by ClarenceThreepwood — APPROVED** [review]

**Comment by pranjalssh** [comment]
We have a session flag isPrintStatsForNonJoinQuery that we should use here(only do this when set).

Also, there is a planStatisticsRead field in QueryCompletedEvent as well. Does that work for your debugging?

**Comment by aaneja** [comment]
Thank you for the pointer @pranjalssh.  
planStatisticsRead & [this commit](https://github.com/prestodb/presto/commit/a67150e1c8f1ea6c037b53a92a879bf328c18183) should suffice for my debugging needs.

Closing this PR

**Pull request closed without merging**